README.md

```diff
@@ -54,18 +54,18 @@ The workflow for creating a new migration is:
 
 #### Frontend Development
 
-To do frontend development you will want to install [`nvm`](https://github.com/creationix/nvm#install-script) and use at least version 6.x. After following the directions above for starting the VM, start the API server and other backend services by running `./scripts/server`.
+To do frontend development you will want to install [`nvm`](https://github.com/creationix/nvm#install-script) and use at least version 6.x. Once using `nvm`, install [yarn](https://yarnpkg.com/) with `npm install -g yarn`. After following the directions above for starting the VM, start the API server and other backend services by running `./scripts/server`.
```

Comment:
While testing I started off using node version 6.1. When I would run `yarn run start` it would fail and shoot an error. I switch that out for version 6.8.1 and that fixed my issue. Might want to change this to say at least 6.8.1